Service/db/init.sql

```diff
@@ -36,8 +36,7 @@ CREATE TABLE Transaction (
    id INT(11) AUTO_INCREMENT PRIMARY KEY,
    sharePrice DECIMAL(13, 2),
    shareCount INT(11),
-   isBuy TINYINT(1),
-   datetime DATETIME,
+   transType VARCHAR(12),
```

Comment:
I think `action` would be a way more standard name here.

Reply:
Ya, didn't wanna change it while I was already in the middle of doing it so I guess I can do it now.